valid-parentheses/SamTheKorean.py

```diff
@@ -0,0 +1,22 @@
+# Time complexity : O(n)
+# Space complexity : O(n)
+class Solution:
+    def isValid(self, s: str) -> bool:
+        parentheses = {"[": "]", "{": "}", "(": ")"}
+        stack = []
+
+        # if ch is opening bracket
+        for ch in s:
+            if ch in parentheses:
+                stack.append(ch)
+            # if ch is closing bracket
+            else:
+                # if closing bracket comes without opening bracket previously
+                if not stack:
+                    return False
+                # if closing bracket doesnt match with the latest type of opening bracket
+                elif ch != parentheses[stack.pop()]:
```

Comment:
nit: 위에서 바로 반환하니까, 굳이 `elif`를 쓸 필요없지 않을까요?

```suggestion
                if ch != parentheses[stack.pop()]:
```